apps/rmh-web/pages/api/get-appointments.ts

```diff
@@ -1,11 +1,54 @@
-import { NextApiRequest, NextApiResponse } from 'next';
-import axios from 'axios';
-import gql from 'graphql-tag';
+import { NextApiRequest, NextApiResponse } from "next";
+import { request as GQLRequest, gql } from "graphql-request";
+import { User } from "@/app/pt-portal/models";
 
-export default async function GET(request: NextApiRequest, response: NextApiResponse) {
-  const GRAPHQL_API_URL_API_KEY = process.env.GRAPHQL_API_URL_API_KEY ? process.env.GRAPHQL_API_URL_API_KEY : '';
-  const GRAPHQL_API_URL = process.env.GRAPHQL_API_URL ? process.env.GRAPHQL_API_URL : ''; 
-  console.log(`API_KEY ${GRAPHQL_API_URL_API_KEY}, GRAPHQL_API_URL ${GRAPHQL_API_URL}`)
+interface Result {
+  getAppointments: {
+    items: User[];
+  };
+}
+
+export default async function GET(
+  request: NextApiRequest,
+  response: NextApiResponse
+) {
+  try {
+    const document = gql`
```

Comment:
we should verify request is valid and send appropiate message and error code if not